scripts/pre_redeploy.sh

```diff
@@ -124,3 +124,10 @@ pushd /etc/openstack_deploy/
     sed -i '/^haproxy_package_state.*/d' user_osa_variables_defaults.yml
     echo "haproxy_package_state: 'latest'" >> user_osa_variables_defaults.yml
 popd
+
+# RLM-1438 patch openstack_hosts fork into 14.7.0 to ensure RLM-322 works
+if [[ ${RPC_TARGET_CHECKOUT} == "r14.7.0" ]]; then
```

Comment:
Do we care about any version lower than 14.7.0?

Reply:
Don't think we're deploying anything older via upgrades and this should be taken care of in 14.8.0.